db-server/index.js

```diff
@@ -50,7 +50,8 @@ function createServer(db) {
   const api = restify.createServer({
     formatters: {
       'application/json; q=0.9': safeJsonFormatter
-    }
+    },
+    maxParamLength: 256
```

Comment:
Careful here.  As noted in [1], we allow email addresses to be up to 255 *unicode characters* long.  If you utf-8 and hexlify a long email address with a lot of high unicode codepoints, it could be substantially longer than 256 hex characters.

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1365486

Reply:
For example:

```
>>> email = u"\N{SNOWMAN}" * 249 + "@x.com"
>>> # The auth-server will accept this as a valid email address
>>> len(email)
255
>>> # But it's very long in byte form!
>>> len(email.encode('utf8').encode('hex'))
1506
```

Reply:
Given current auth-server limits (no astral-plane characters, 255 code-points) I think a safe upper bound would be `255 * 3 * 2 = 1530` hex-encoded utf8 bytes.

Reply:
@rfk Good call, didn't think about that. I updated accordingly and added some notes.